mesonbuild/backend/vs2010backend.py

```diff
@@ -1176,9 +1174,8 @@ def gen_testproj(self, target_name, ofname):
         postbuild = ET.SubElement(action, 'PostBuildEvent')
         ET.SubElement(postbuild, 'Message')
         # FIXME: No benchmarks?
-        test_command = [sys.executable,
-                        self.environment.get_build_command(),
-                        'test',
+        test_command = self.environment.get_build_command() + \
+                        ['test',
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->